cmd/gcp-controller-manager/csr_signer.go

```diff
@@ -64,38 +69,58 @@ func newGKESigner(ctx *controllerContext) (*gkeSigner, error) {
 		ctx:          ctx,
 		retryBackoff: ClusterSigningGKERetryBackoff,
 		validators:   csrValidators(ctx),
+
+		csrs: csrs,
 	}, nil
 }
 
+// handle is called by the generic Kubernetes certificate controller.
 func (s *gkeSigner) handle(csr *capi.CertificateSigningRequest) error {
+	_, _, err := s.handleInternal(csr)
+	return err
+}
+
+func (s *gkeSigner) handleInternal(csr *capi.CertificateSigningRequest) (processed bool, out *capi.CertificateSigningRequest, err error) {
```

Comment:
These new return values aren't used ever?

Reply:
They are used for testing; previously we were only testing the `sign` function, so half of this file didn't have coverage.